Sources/Client/Client/Endpoint.swift

```diff
@@ -54,9 +54,9 @@ public enum Endpoint {
     /// Send a message to a channel.
     case sendMessage(Message, Channel)
     /// Upload an image to a channel.
-    case sendImage(_ fileName: String, _ mimeType: String, Data, Channel)
+    case sendImage(Data, _ fileName: String, _ mimeType: String, Channel)
     /// Upload a file to a channel.
-    case sendFile(_ fileName: String, _ mimeType: String, Data, Channel)
+    case sendFile(Data, _ fileName: String, _ mimeType: String, Channel)
```

Comment:
The same as the above. We should make `Endpoint` internal because we provide API for endpoints with a specific callback type without this type the request with some endpoint useless, you need to know response type.